client/web/src/site-admin/SiteAdminWebhooksPage.story.tsx

```diff
@@ -0,0 +1,124 @@
+                                        },
+                                    },
+                                },
+                            },
+                        },
+                    ])
+                }
+            >
+                <SiteAdminWebhooksPage
+                    match={{} as any}
+                    history={H.createMemoryHistory()}
+                    location={{} as any}
+                    telemetryService={NOOP_TELEMETRY_SERVICE}
+                />
+            </MockedTestProvider>
+        )}
+    </WebStory>
+)
+
+NoWebhooksFound.storyName = 'No webhooks found'
+
+export const FiveWebhooksFound: Story = () => (
+    <WebStory>
+        {() => (
+            <MockedTestProvider
+                link={
+                    new WildcardMockLink([
+                        {
+                            request: {
+                                query: getDocumentNode(WEBHOOKS),
+                                variables: MATCH_ANY_PARAMETERS,
+                            },
+                            result: {
+                                data: {
+                                    webhooks: {
+                                        nodes: [
+                                            createWebhookMock(
+                                                ExternalServiceKind.BITBUCKETCLOUD,
+                                                'bitbucket.com/repo1'
+                                            ),
+                                            createWebhookMock(ExternalServiceKind.GITHUB, 'github.com/repo1'),
+                                            createWebhookMock(ExternalServiceKind.GITHUB, 'github.com/repo2'),
+                                            createWebhookMock(ExternalServiceKind.GITHUB, 'github.com/repo3'),
+                                            createWebhookMock(ExternalServiceKind.BITBUCKETCLOUD, 'github.com/repo2'),
+                                        ],
+                                        totalCount: 5,
+                                        pageInfo: {
+                                            hasNextPage: false,
+                                        },
+                                    },
+                                },
+                            },
+                        },
+                    ])
+                }
+            >
+                <SiteAdminWebhooksPage
+                    match={{} as any}
+                    history={H.createMemoryHistory()}
+                    location={{} as any}
+                    telemetryService={NOOP_TELEMETRY_SERVICE}
+                />
+            </MockedTestProvider>
+        )}
+    </WebStory>
+)
+
+FiveWebhooksFound.storyName = '5 webhooks found'
+
+function createWebhookMock(kind: ExternalServiceKind, urn: string): WebhookFields {
+    return {
+        __typename: 'Webhook',
+        createdAt: '',
+        id: '',
```

Comment:
I think you need to give them an `id` so they're not treated as the same node by `useConnection`.

```suggestion
        id: `webhook-${urn}`,
```

That fixes the list for me.

Reply:
that's absolutely it, thank you!
will fix it and then merge the PR.

Thanks for your review!